editor/store-persist.js

```diff
@@ -1,19 +1,28 @@
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { STORE_DEFAULTS } from './store-defaults';
+import { identity } from 'lodash';
 
+/**
+ * Internal dependencies
+ */
 const DEFAULT_STORAGE_KEY = 'REDUX_PERSIST';
 
 /**
  * Store enhancer to persist a specified reducer key
- * @param {String}     reducerKey The reducer key to persist
- * @param {String}     storageKey The storage key to use
- * @param {Object}     defaults   Default values
+ * @param {String}     reducerKey          The reducer key to persist
```

Comment:
I'm surprised this is not being flagged by the [`valid-jsdoc` ESLint rule](https://eslint.org/docs/rules/valid-jsdoc), since this function now has only a single parameter.

There are some ways to document expected object properties, outlined here:

http://usejsdoc.org/tags-type.html